python/peacock/tests/exodus_tab/tests

```diff
@@ -1,93 +1,110 @@
 [Tests]
+  issues = '#7451'
+  design = 'python/peacock.md'
   [VTKWindowPlugin]
     type = PythonUnitTest
     input = test_VTKWindowPlugin.py
     separate = True
     display_required = true
+    requirement = "The system shall include a graphical interface that can display both mesh and solution Exodus files and let the user, including series of files created with mesh adaptivity."
```

Comment:
Let the user what?

Reply:
Added a list.
These tests are very not-granular. I m sure it saved some time building the test suite but it makes for off requirements